src/webgpu/api/validation/encoding/encoder_state.spec.ts

```diff
@@ -52,8 +52,6 @@ g.test('pass_end_invalid_order')
     `
   Test that beginning a {compute,render} pass before ending the previous {compute,render} pass
   causes an error.
-
-  TODO: Update this test according to https://github.com/gpuweb/gpuweb/issues/2464
   `
   )
   .params(u =>
```

Comment:
I think it would be equivalent and simpler to modify the subcases instead:

```
      // Don't end the first pass multiple times (that generates a validation error but doesn't invalidate the encoder)
      .unless(p => p.firstPassEnd && p.endPasses.includes(0))
```

Reply:
Good point. Done!